spec/nested_spec.js

```diff
@@ -28,4 +28,16 @@ describe('Frisby nested calls', function() {
       });
   });
 
+  it('should allow provide previous requests response header', function(doneFn) {
+    mocks.use(['provideHeaders']);
+
+    // Fetch headers
+    frisby.fetch(testHost + '/provideheaders')
+      .expect('status', 200)
+      .then(function(body, headers) {
```

Comment:
You should be able to just use `expect('header', 'Some-Headers', 'Some-Value')` here instead of adding another parameter here. Promises traditionally only take a single parameter, so I don't want to break that expectation.